### ISSUE TITLE
Configurable number of detonations?

### ISSUE THREAD

**Issue by griffenx** [Europia79/Demolition#5]
I had an idea for the plugin. It'd be nice to be able to toggle the number of detonations required to win a match. I.e. for BombArena matches, each team needs 3 detonations to win without counting defusings. I think it'd be nice to be able to allow server owners to customize how long matches last.

**Comment by Europia79** [comment]
Yeah, I agree... it sounds exactly like my idea for BattleArena to have multiple rounds...

In your example, it would be best of five rounds... 
And the first one to win three rounds would win the match.
